crates/mysticeti-core/src/consensus/base_committer.rs

```diff
@@ -160,22 +169,22 @@ impl BaseCommitter {
 
     /// Check whether the specified leader has enough blames (that is, 4f+1 non-supports) to be
     /// directly skipped.
-    fn enough_leader_blame(&self, decision_round: RoundNumber, leader: AuthorityIndex) -> bool {
-        let decision_blocks = self.block_store.get_blocks_by_round(decision_round);
+    fn enough_leader_blame(&self, voting_round: RoundNumber, leader: AuthorityIndex) -> bool {
+        let voting_blocks = self.block_store.get_blocks_by_round(voting_round);
 
         let mut blame_stake_aggregator = StakeAggregator::<QuorumThreshold>::new();
-        for decision_block in &decision_blocks {
-            let decider = decision_block.reference().authority;
-            if decision_block
+        for voting_block in &voting_blocks {
+            let voter = voting_block.reference().authority;
+            if voting_block
                 .includes()
                 .iter()
                 .all(|include| include.authority != leader)
             {
                 tracing::trace!(
-                    "[{self}] {decision_block:?} is a blame for leader {}",
-                    format_authority_round(leader, decision_round - 1)
+                    "[{self}] {voting_block:?} is a blame for leader {}",
+                    format_authority_round(leader, voting_round - 1)
                 );
```

Comment:
let's drop these trace if not necessary (they shoulnd' have been there in the first place)